int32[11] KEY_CODE
